.dockerignore

```diff
@@ -0,0 +1,30 @@
+# Python
+__pycache__
+*.pyc
+**/__pycache__
+**/*.pyc
+build/
+dist/
+**/*.egg-info
+venv
+.venv
```

Comment:
you should also add

```
**/venv
**/.venv
```

Reply:
Do you think people are willing to create virtualenvs everywhere in the project? 

![](https://media2.giphy.com/media/MwOuiiTfWfWgM/giphy.gif)